lib/matplotlib/style/core.py

```diff
@@ -65,6 +65,14 @@ def _apply_style(d, warn=True):
     mpl.rcParams.update(_remove_blacklisted_style_params(d, warn=warn))
 
 
+@docstring.Substitution(
+    "\n".join(
+        sorted(
+            set(map("- {}".format, map(str.strip, STYLE_BLACKLIST))),
```

Comment:
`STYLE_BLACKLIST` is a `set` already, no need to wrap it.
```suggestion
            map("- {}".format, map(str.strip, STYLE_BLACKLIST)),
```

Reply:
I used the `set` to remove possible duplicates in `STYLE_BLACKLIST`. It's just a security layer in case someone puts a duplicate entry in `STYLE_BLACKLIST` by accident, although this is very unlikely to happen. So, I will remove it.

Reply:
`STYLE_BLACKLIST` is already a set literal when it is defined

https://github.com/matplotlib/matplotlib/blob/586fcffaae03e40f851c5bc854de290b89bae18e/lib/matplotlib/style/core.py#L44-L49

and `sorted` will happily iterate over the map without having to fully materialize it first.  I also agree it is reasonable to expect that it will contain only valid rcparams (which at this point none of them contain whitespace!) and not worry about doing any cleaning.

That said, I am :+1: on the concern of covering surprising inputs graceful!

Reply:
Removed `set` and `str.strip`, kept `sorted`